src/array_buffer.rs

```diff
@@ -175,6 +192,17 @@ impl ArrayBuffer {
     unsafe { scope.to_local(ptr) }.unwrap()
   }
 
+  pub fn new_with_unique_backing_store<'sc>(
+    scope: &mut impl ToLocal<'sc>,
+    backing_store: &mut UniqueRef<BackingStore>,
```

Comment:
This should not be an &mut UniqueRef. 
When you call this function that means that you're going to share the BackingStore with v8 so the user can't hold on to a unique reference.
Instead they can call the `.get_backing_store()` method on the returned `Local<ArrayBuffer>` and get a `SharedRef<BackingStore>` from it.

Reply:
I failed to obtain `SharedRef` from it... I'll give it another go tomorrow